config/airflow.dev.cfg

```diff
@@ -6,6 +6,7 @@ destination_bucket = dev-jats-ingester-expanded
 base_url = https://%(destination_bucket)s.s3.amazonaws.com
 service_name = content-store
 service_url = http://content-store
+temp_directory =
```

Comment:
we probably won't even mount a volume for this directory as a start, but will configure the `/tmp` directory inside the container

Reply:
Should this still be a configurable option or should I just use `/tmp` all the time?